packages/heroku.rb

```diff
@@ -3,15 +3,27 @@
 class Heroku < Package
   description 'The Heroku Command Line Interface (CLI), formerly known as the Heroku Toolbelt, is a tool for creating and managing Heroku apps from the command line / shell of various operating systems.'
   homepage 'https://devcenter.heroku.com/articles/heroku-cli'
-  version '4.28.3'
-  binary_url ({
-    armv7l: "https://drive.google.com/uc?export=download&id=0ByCixsDmZPzxOHFIMzQyNHNFUWc",
-    i686: "https://drive.google.com/uc?export=download&id=0ByCixsDmZPzxd3NULTRkMWlHQTA",
-    x86_64: "https://drive.google.com/uc?export=download&id=0ByCixsDmZPzxLURkMktpREpDZk0"
-  })
-  binary_sha256 ({
-    armv7l: "b7fab53d3c8cfd42d41c1b07db069c9fd5f7261fcea48fd99114981cf1a293d7",
-    i686: "1e73fbff3a10c0c95dd1aa1cf68952a95a330634fa410765962df0396315a42c",
-    x86_64: "9259cd8b5d7aaeb9172142b5956ad767a3d4f1bcf126e51a7f4a8c055e53c068"
-  })
+  version '6.13.6'
+  source_url 'https://github.com/heroku/cli/archive/v6.13.6.tar.gz'
+  source_sha256 '395c7e22d16b1fce2d5ed4ac1e022dbd3c9aa8fdbc172d9dadf86ba9c1ad6978'
+
+  depends_on 'nvm'
+  depends_on 'yarn'
+
+  def self.build
+    system '. ~/.bashrc && nvm install stable && yarn install'
+  end
+
+  def self.install
+    system "mkdir -p #{CREW_DEST_DIR}#{CREW_PREFIX}/bin"
+    system "mkdir -p #{CREW_DEST_DIR}#{CREW_PREFIX}/share/heroku"
+    system "cp -r . #{CREW_DEST_DIR}#{CREW_PREFIX}/share/heroku"
+    FileUtils.cd("#{CREW_DEST_DIR}#{CREW_PREFIX}/bin") do
+      system "ln -s /usr/local/share/heroku/bin/run heroku"
+    end
+    puts ""
+    puts "To finish the installation, execute the following:".lightblue
+    puts "source ~/.bashrc".lightblue
+    puts ""
```

Comment:
Although I agree we shouldn't force the user to install node in any particular manner, there are several issues with this line:
```
depends_on 'node' unless `node --version`
```
1. We need node version 8 and `node` installs version 6.  We should instead determine what version is currently installed and suggest the user install (or better yet, install automatically) a more updated version that meets the minimum requirement.
2. `node --version` does not work when `nvm` is installed unless `~/.bashrc` is sourced first.
3. Compiling node from source takes an eternity on a slow Chromebook (like over 2 hours).

I would recommend we use nvm or nodebrew instead of compiling from source.  I'd like to update the `yarn` package to install this way eventually also.